index.html

```diff
@@ -80,11 +80,47 @@ <h4>Island Getaway</h4>
         <div class="btn">Sign Me Up!</div>
       </div>
     </section>
+    <form class="form" action="" method="post">
```

Comment:
I like the form (I think you're reusing it from the past example) but it's a nice form. Like the use of fieldset.